### PR TITLE
Add checkpointing with TF/conda

### PR DESCRIPTION
**Pull request by jhiemstrawisc**
Add example of checkpointing with TF in a conda environment, using the gpu-examples squid server

### CODE REVIEW

File: checkpointing/conda/tensorflow/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+## Checkpointing in Tensorflow with conda environment and GPU support
+The example uses a pre-configured conda environment to demonstrate how one might implement model checkpointing in Tensorflow.
+
+The Tensorflow example downloads the MNIST dataset and trains an neural net on it for 20 epochs, making a "checkpoint" every fifth epoch. In the event of a runtime interruption, these checkpoints are returned to the user and can be used to resume training, preventing the loss of already-completed work.
+
+The four files include:
+- checkpoint.sub ## used to submit the job on HTCondor
```
Comment (ChristinaLK) [inline checkpointing/conda/tensorflow/README.md:7]:
```suggestion
- `checkpoint.sub` ## used to submit the job on HTCondor
```

Comment (ChristinaLK) [reply checkpointing/conda/tensorflow/README.md:7]: I would make these changes for the remaining bullets also.

**Review by ChristinaLK — COMMENTED** [review]
Partial review, Pat should also review.

File: checkpointing/conda/tensorflow/checkpoint.sub [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+arguments = $(Process)
+output = out.out
+error = err.err
+
+should_transfer_files = YES
+when_to_transfer_output = ON_EXIT
+transfer_input_files = tf_checkpointing.py,http://proxy.chtc.wisc.edu/SQUID/gpu-examples/tf_checkpointing.tar.gz 
+
+
+requirements = (Target.CUDADriverVersion >= 10.1)
```
Comment (PatStiles) [inline checkpointing/conda/tensorflow/checkpoint.sub:19]:
```suggestion
requirements = (CUDADriverVersion >= 10.1)
```

**Review by PatStiles — COMMENTED** [review]
Perhaps alter the file structure the templates are located within the checkpointing folder i.e. templates-GPUs/checkpointing/tf_checkpointing vs. templates-GPUs/conda/tensorflow. Would welcom peoples thoughts.

**Comment by ChristinaLK** [comment]
Changes made in #20

**Pull request closed without merging**